docs/production/mobile-push-notifications.md

```diff
@@ -2,64 +2,65 @@
 
 Zulip's iOS and Android mobile apps support receiving push
 notifications from Zulip servers to let users know when new messages
-have arrived.  This is an important feature to having a great
+have arrived. This is an important feature to having a great
 experience using the Zulip mobile apps.
 
 For technical reasons (explained below), in order to deliver mobile
 push notifications in the app store versions of our mobile apps, you
 will need to register your Zulip server with the Zulip mobile push
-notification service.  This service will forward push notifications
+notification service. This service will forward push notifications
 generated by your server to the Zulip mobile app automatically.
 
 ## How to sign up
 
 Starting with Zulip 1.6 for both Android and iOS, Zulip servers
 support forwarding push notifications to a central push notification
-forwarding service.  You can enable this for your Zulip server as
+forwarding service. You can enable this for your Zulip server as
```

Comment:
Can you please remove the spurious changes to documentation that you're not modifying?  It makes your changes very hard to review.

Reply:
Thanks for mentioning it. I have removed the spurious changes to the file which were brought in due to linting in my VSCode.